### ISSUE TITLE
Solr 4.4 *_geom issue

### ISSUE THREAD

**Issue by waynegraham** [geoblacklight/geoblacklight-schema#27]
I am trying to post Solr docs to a Solr 4.4 instance. In 4.10.x, they post fine, but in 4.4, I get

```xml
<?xml version="1.0" encoding="UTF-8"?>
<response>
<lst name="responseHeader"><int name="status">400</int><int name="QTime">2</int></lst><lst name="error"><str name="msg">com.spatial4j.core.exception.InvalidShapeException: Unable to read: ENVELOPE(-78.83913, -78.20801, 38.27794, 37.97745)</str><int name="code">400</int></lst>
</response>
```

Before I dug too far in to this, I wanted to check to see if anyone knew if there was something weird about this particular version of Solr.

Thanks!

**Comment by mejackreed** [comment]
@waynegraham the `solrConfig` https://github.com/geoblacklight/geoblacklight-schema/blob/master/conf/solrconfig.xml#L23 we use specifies at least version 4.7. I know Solr changed a lot of spatial stuff from 4.0 - 4.10 so that may be an issue. Just to check, do you have the `location_rpt` type configured in your Schema?

**Comment by waynegraham** [comment]
Yeah, the only difference between the config I have and the master branch is changing the least version of Lucene (it was changed to 4.4). I'll dive in to this a bit more, if I find anything out I'll post here.

**Comment by drh-stanford** [comment]
Yes, you need Solr 4.7 or later to support the `ENVELOPE` syntax. They upgraded to Spatial4J 0.4 which has the `ENVELOPE` syntax support.

See http://lucene.apache.org/solr/4_7_0/changes/Changes.html#v4.7.0.new_features and https://github.com/spatial4j/spatial4j/blob/master/CHANGES.md

**Comment by waynegraham** [comment]
Thanks! I had just found a reference to https://issues.apache.org/jira/browse/LUCENE-5395 in the change log which got fixed in 4.7.x. Now to see if I can get the server admins to update that
